test/lit/passes/simplify-locals-desc.wast

```diff
@@ -1,31 +0,0 @@
-;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
```

Comment:
Why delete this?

Reply:
This was just testing the effects for `struct.new`. We do so much more thoroughly and directly in the new test.